en/partial-updates.html

```diff
@@ -149,9 +163,17 @@ <h2 id="write-pipeline">Write pipeline</h2>
       More details in <a href="attributes.html">attributes</a>.
     </p>
     <p>
-      Note: if the attribute field is an array of map or struct,
-      values are written in the summary store and update rates hence lower -
-      refer to <a href="https://github.com/vespa-engine/vespa/issues/10892">#10892</a> updates of array of map/struct.
+      Updating <a href="schemas.html#multivalue-fields">multivalue</a> attributes (arrays, maps, sets, tensors)
+      means reading the current value, making the update and writing it back:
+      <ul>
+        <li><a href="reference/schema-reference.html#type:weightedset">Weightedsets</a> and
+        <a href="reference/schema-reference.html#type:tensor">tensors</a> are in memory and therefore fast,
+        see <a href="attributes.html#data-structures">attribute data structures</a> for performance considerations.</li>
+        <li>If the attribute field is an <a href="reference/schema-reference.html#type:array">array of map or struct</a>,
```

Comment:
Rename to "array of struct" (with current link) or "map" (with another link: reference/schema-reference.html#type:map).